docs/testnet/join-testnet-tutorial.md

```diff
@@ -0,0 +1,292 @@
+```
+# Retrieve public ip address
+MY_IP=$(host -4 myip.opendns.com resolver1.opendns.com | grep "address" | awk '{print $4}')
+
+# Get persistent peer
+COORDINATOR_P2P_ADDRESS=$(jq -r '.app_state.genutil.gen_txs[0].body.memo' <prov-node-dir>/config/genesis.json)
+
+# Run node
+interchain-security-pd start --home <prov-node-dir> \
+        --rpc.laddr tcp://${MY_IP}:26658 \
+        --grpc.address ${MY_IP}:9091 \
+        --address tcp://${MY_IP}:26655 \
+        --p2p.laddr tcp://${MY_IP}:26656 \
+        --grpc-web.enable=false \
+        --p2p.persistent_peers $COORDINATOR_P2P_ADDRESS \
+        &> <prov-node-dir>/logs &
+```
+   
+   
+* *If you get the error "can't bind address xxx.xxx.x.x", try using `127.0.0.1` instead.*   
+
+
+* *If you're running the coordinator node on the same local machine, you might
+need to change its IP address to 127.0.0.1.*   
+
+
+<br/><br/>
+
+__6. Fund your account__   
+Make sure your node account has at least `1000000stake` coins in order to stake.
+Verify your account balance using the command below.
+```
+# Setup the client RPC endpoint using the following command.
+sed -i -r "/node =/ s/= .*/= \"tcp:\/\/${MY_IP}:26658\"/" <prov-node-dir>/config/client.toml
+
+# Check your account balance
+interchain-security-pd q bank balances $(jq -r .address <prov-node-dir>/<provider-keyname_keypair>.json) --home <prov-node-dir>
+```
+
+* *Ask to get your local account fauceted or use the command below if you have access to another account at least extra `1000000stake` tokens.*
+
+ ```
+# Get local account addresses
+ACCOUNT_ADDR=$(interchain-security-pd keys show <your-keyname> \
+       --keyring-backend test --home <prov-node-dir> --output json | jq -r '.address')
+
+# Run this command 
+interchain-security-pd tx bank send <source-address> <destination-address> \
+        1000000stake --from <source-keyname> --keyring-backend test --home <prov-node-dir> --chain-id provider -b block 
+```
+
+<br/><br/>
+  
+__7. Bond the validator__  
+Bond your local node as a validator by sending the following transaction.
+
+```
+# Get the validator node pubkey 
+VAL_PUBKEY=$(interchain-security-pd tendermint show-validator --home <prov-node-dir>)
+
+# Create the validator
+interchain-security-pd tx staking create-validator \
+            --amount 1000000stake \
+            --pubkey $VAL_PUBKEY \
+            --from <provider-keyname> \
+            --keyring-backend test \
+            --home <prov-node-dir> \
+            --chain-id provider \
+            --commission-max-change-rate 0.01 \
+            --commission-max-rate 0.2 \
+            --commission-rate 0.1 \
+            --moniker <prov-node-moniker> \
+            --min-self-delegation 1 \
+            -b block -y
+```
+<br>
+Verify that your validator node is now part of the validator-set.
+
+```
+interchain-security-pd q tendermint-validator-set --home <prov-node-dir>
+```  
+
+---
+
+### Run a validator on the Consumer chain
```

Comment:
Can we do this step first? Like this we avoid getting slashed for downtime if setting up the consumer validator takes too long.